### PR TITLE
[VEN-1936]: View functions to getAmountIn and getAmountOut

### PR DESCRIPTION
**Pull request by Debugger022**
This PR contains getter functions for **getAmountIn** and **getAmountOut** in **AbstractTokenConverter**.

### CODE REVIEW

File: contracts/TokenConverter/AbstractTokenConverter.sol [hunk 1]
```diff
@@ -357,11 +357,44 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
...
+    /// @custom:error InsufficientInputAmount error is thrown when given input amount is zero
+    /// @custom:error ConversionConfigNotEnabled is thrown when convert is disabled or config does not exist for given pair
+    function getUpdatedAmountOut(
+        uint256 amountInMantissa,
+        address tokenAddressIn,
+        address tokenAddressOut
+    ) public returns (uint256 amountConvertedMantissa, uint256 amountOutMantissa) {
+        priceOracle.updateAssetPrice(tokenAddressIn);
+        priceOracle.updateAssetPrice(tokenAddressOut);
+        return getAmountOut(amountInMantissa, tokenAddressIn, tokenAddressOut);
```
Comment (chechu) [inline contracts/TokenConverter/AbstractTokenConverter.sol:375]:
```suggestion
        (amountConvertedMantissa,  amountOutMantissa ) = getAmountOut(amountInMantissa, tokenAddressIn, tokenAddressOut);
```

File: contracts/TokenConverter/AbstractTokenConverter.sol [hunk 2]
```diff
@@ -357,11 +357,44 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
...
+    /// @custom:error InsufficientInputAmount error is thrown when given input amount is zero
+    /// @custom:error ConversionConfigNotEnabled is thrown when convert is disabled or config does not exist for given pair
+    function getUpdatedAmountIn(
+        uint256 amountOutMantissa,
+        address tokenAddressIn,
+        address tokenAddressOut
+    ) public returns (uint256 amountConvertedMantissa, uint256 amountInMantissa) {
+        priceOracle.updateAssetPrice(tokenAddressIn);
+        priceOracle.updateAssetPrice(tokenAddressOut);
+        return getAmountIn(amountOutMantissa, tokenAddressIn, tokenAddressOut);
```
Comment (chechu) [inline contracts/TokenConverter/AbstractTokenConverter.sol:393]:
```suggestion
        (amountConvertedMantissa,  amountInMantissa ) = getAmountIn(amountOutMantissa, tokenAddressIn, tokenAddressOut);
```

Comment (Debugger022) [reply contracts/TokenConverter/AbstractTokenConverter.sol]: Done [a0d3f2a](https://github.com/VenusProtocol/protocol-reserve/pull/19/commits/a0d3f2ad5fd685b55c692545beb29c387059bc87)

File: contracts/TokenConverter/AbstractTokenConverter.sol (hunk 1 above)
Comment (Debugger022) [reply contracts/TokenConverter/AbstractTokenConverter.sol]: Done [a0d3f2a](https://github.com/VenusProtocol/protocol-reserve/pull/19/commits/a0d3f2ad5fd685b55c692545beb29c387059bc87)

**Pull request merged**